apps/omg_status/lib/omg_status/metric/statsd_monitor.ex

```diff
@@ -120,10 +120,10 @@ defmodule OMG.Status.Metric.StatsdMonitor do
   defp raise_clear(_alarm_module, true, false), do: :ok
 
   defp raise_clear(alarm_module, false, false),
-    do: alarm_module.set({:statsd_client_connection, Node.self(), __MODULE__})
+    do: alarm_module.set(alarm_module.statsd_client_connection(__MODULE__))
```

Comment:
`alarm_module` could be omitted at `init()`. Let's replace:
`alarm_module = Keyword.get(opts, :alarm_module)` with: 
`alarm_module = Keyword.fetch!(opts, :alarm_module)`

Reply:
I guess `child_module` could be made required the same way

Reply:
I'm not exactly sure I understand what you mean by this comment... could you elaborate pls?

Reply:
Jumping in (Papa is out). I think this is meant: https://github.com/omisego/elixir-omg/blob/8795fb527d3eb4c347142fb4e1b26faa7f380ff3/apps/omg_status/lib/omg_status/metric/statsd_monitor.ex#L48

`get` is used there, so this can be `nil` and the suggestion is to `fetch!` there to avoid a `nil.statsd_client_connection` and fail fast instead.